mibs/gigabyte-bmc/gigabyte-bmc

```diff
@@ -0,0 +1,748 @@
+---------------------------------------------------------------------------
```

Comment:
The MIB should be in a directory called gigabyte and the file name should be GIGABYTE-BMC

Reply:
Done.